src/index.ts

```diff
@@ -108,7 +101,7 @@ getUserFunction(CODE_LOCATION, TARGET).then(userFunction => {
 
   SERVER.listen(PORT, () => {
     ERROR_HANDLER.register();
-    if (process.env.NODE_ENV !== NodeEnv.PRODUCTION) {
+    if (process.env.NODE_ENV !== 'production') {
```

Comment:
I'm not seeing TS error for misconfigured strings with all.

For example, if I type:
```ts
if (process.env.NODE_ENV !== 'prodduction') {
```

here, I won't get a TS error. I'd rather just be able to type `NodeEnv.` and see an autocompleted result of options and errors if I mistype.